### ISSUE TITLE
Error: Failed to instantiate provider "virtualbox" to obtain schema: Incompatible API version with plugin. Plugin version: 4, Client versions: [5]

### ISSUE THREAD

**Issue by basictheprogram** [terra-farm/terraform-provider-virtualbox#59]
```
$ terraform --version
Terraform v0.12.3
+ provider.virtualbox (unversioned)
```
## main.tf
```go
resource "virtualbox_vm" "node" {
    count = 2
    name = "${format("node-%02d", count.index+1)}"
    url = "https://vagrantcloud.com/ubuntu/boxes/xenial64/versions/20180420.0.0/providers/virtualbox.box"
    image = "./virtualbox.box"
    cpus = 2

     network_adapter {
       type = "bridged"
       host_interface="en0"
    }
}
```
```
$ terraform init
```
```
$ terraform plan

Error: Failed to instantiate provider "virtualbox" to obtain schema: Incompatible API version with plugin. Plugin version: 4, Client versions: [5]
```

**Comment by VoyTechnology** [comment]
It looks like it might be related to the provider using an older version of Terraform:
https://github.com/terra-farm/terraform-provider-virtualbox/blob/c4fcdba845e54a69638f7d6ba34037caf8b6246f/go.mod#L8

This would be part of #50 to get it fixed.

**Comment by VoyTechnology** [comment]
@basictheprogram You also are using both url and image, these two fields have been merged. Please take a look at the README. It would turn into

```
image = "https://vagrantcloud.com/ubuntu/boxes/xenial64/versions/20180420.0.0/providers/virtualbox.box"
```

This is probably not related, just a FYI.

**Comment by ringods** [comment]
@basictheprogram we have released a TF 0.12 compatible version. You can download it here:

https://github.com/terra-farm/terraform-provider-virtualbox/releases/tag/v0.2.0

**Comment by abhi-vaidya** [comment]
Any plans to make this plugin available for windows ?
https://github.com/terra-farm/terraform-provider-virtualbox/releases/tag/v0.2.0
